values.yaml

```diff
@@ -107,6 +109,23 @@ namespaces:
   ingress:
     enabled: true
 
+# Proxy Node namespaces
+
+# TODO: Need to use the master branch VMC for the new PN
```

Comment:
I'm not sure I understand what this comment means?

Reply:
I thought maybe we'll need to move the current VMC to a legacy branch as well and have a new deployment for the multi-country PN but maybe that's not the case. @idrop is looking into this